Sources/GuitarCharacter.swift

```diff
@@ -36,6 +36,18 @@ public extension String {
         return reversed().first()
     }
 
+    /// Returns the latinized version of the string without diacritics.
+    ///
+    ///     let string = "Hello! こんにちは! สวัสดี! مرحبا! 您好!"
+    ///     print(string.latinized())
+    ///     // Prints "Hello! kon'nichiha! swasdi! mrhba! nin hao!"
+    ///
+    /// - Returns: The latinized version of the string without diacritics.
+    @discardableResult
+    func latinized() -> String {
+        return (applyingTransform(.toLatin, reverse: false) ?? self).withoutAccents()
```

Comment:
`String.applyingTransform` returns an optional, so I had to use `??` and just defaulted to returning self if for some reason it's nil